app/renderer/components/add_task.js

```diff
@@ -3,11 +3,13 @@
 const TaskInput = require('./task_input');
 
 module.exports = {
+    props: ['tasks'],
     data() {
         return {
             title: "",
             command: "",
             path: "",
+            order: 0,
```

Comment:
Do we really need the order explicitly inside the task? the order should only affect to suites, and the array of tasks already contains that information